.reuse/dep5

```diff
@@ -4,12 +4,12 @@ Upstream-Contact:  <info@kdab.com>
 Source: https://www.github.com/KDAB/KDBindings
 
 Files: *.json *.html *.md docs/api/Doxyfile.cmake docs/api/mkdocs/mkdocs.yml docs/api/mkdocs/docs/stylesheets/kdab.css
-Copyright: 2020-2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+Copyright: 2020-2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
Why does this still include the current year?
Ideally we'd want to use the year of creation here as well I think as then we don't need to update the files at all.

Same with mkdocs.yml.cmake.